pwndbg/color/syntax_highlight.py

```diff
@@ -71,6 +72,11 @@ def syntax_highlight(code, filename='.asm'):
 
     if lexer:
         lexer_cache[filename] = lexer
+
+        # Count the number of newlines at start of the code
+        # As for some reason pygments.highlight remove them, we need to bring them back
+        # So that if we use this later for displaying code+code lines, they match
+        newlines_at_start = sum(1 for _ in itertools.takewhile(lambda c: c == '\n', code))
         code = pygments.highlight(code, lexer, formatter).rstrip()
 
-    return code
+    return '\n' * newlines_at_start + code
```

Comment:
Instead of using itertools with this, I think we should use stripnl=False when creating the lexer to fix this problem. (http://pygments.org/docs/lexers/)

Reply:
That works perfectly, thanks!